driver/common.env

```diff
@@ -3,4 +3,3 @@ NETWORK_ID=5777
 PRIVATE_KEY=4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d
 SOLVER_TYPE=naive-solver
 SCHEDULER=evm
-GAS_PRICE_CAP_SUBSIDY_FACTOR=100
```

Comment:
@e00E Is it OK to remove this? And if not should the default value be updated in `driver/src/main.rs`?